projects/components/src/utils/test/widget-object/cypress/cypress-widget-finder.ts

```diff
@@ -40,11 +40,23 @@ export class CypressWidgetObjectFinder<T> {
         if (findOptions?.cssSelector) {
             tagName += `${findOptions.cssSelector}`;
         }
+
         const id = idGenerator.generate();
-        const search = findOptions?.ancestor
-            ? cy.get(findOptions.ancestor).find(tagName)
-            : cy.get(tagName, findOptions?.options);
-        const root = search.as(id);
+
+        const ancestor = findOptions?.ancestor ? cy.get(findOptions?.ancestor) : cy.get('body');
+        const parentWidget = new CypressWidgetObjectElement(ancestor, false, undefined);
+        let query = widgetConstructor.tagName;
+        if (findOptions?.cssSelector) {
+            query = query + findOptions.cssSelector;
```

Comment:
Don't you need a space when combining selectors?

Reply:
No, the selector should be applied to the root element, it's combining it with the tag name? I believe, @ryan-bradford ?

Reply:
Yea, there's a case where you want to get some tagName with a specific class applied (`vcd-data-exporter.vms-exporter`), and for this case, we can't have a space.